scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -271,16 +271,30 @@
   </div>
   <div class="row">
     <div class="col-2">
-      <a class="btn btn-secondary btn-sm" data-toggle="collapse" href="#stringcoords" aria-expanded="false" aria-controls="stringcoords">
+      <a class="btn btn-secondary btn-sm form-control" data-toggle="collapse" href="#stringcoords" aria-expanded="false" aria-controls="stringcoords">
         String coordinates
       </a>
     </div>
-    <div class="col-3 offset-1">
-      <div id="stringcoords" class="collapse">
-        {{ wtf.form_field(form.chrom_pos) }}
+    <div class="col-4">
```

Comment:
<img width="491" alt="image" src="https://user-images.githubusercontent.com/28093618/158977956-9f7c3908-22d4-46e1-bc79-13bc195d97be.png">

Reply:
This part is a bit crammed, perhaps give more space to Local obs by reducing the space occupied by Chromosome?

Reply:
You are too zoomed in! The local obs box already looks weird! 😄 Perhaps I can fix it as well!

Reply:
I'm zoomed at 100% on safari and chrome, but perhaps's it's because I'm using my laptop and not a monitor?

Reply:
Probably! Its going to be the same for many - I've tweaked it a little in the last push! See if you like it! I made the String button the same size as the Hide to retain color symmetry on the page. It would look a little lopsided if they are different.

Reply:
Looking good now:

<img width="722" alt="image" src="https://user-images.githubusercontent.com/28093618/159216345-c9313676-b3a6-49cc-9958-1f86ee7ca896.png">